.pre-commit-config.yaml

```diff
@@ -26,7 +26,7 @@ repos:
       hooks:
           - id: black
     - repo: https://github.com/pre-commit/mirrors-mypy
-      rev: v0.990
+      rev: v1.1.1
```

Comment:
You could make a PR  to `main` with just 169b9fa81bd5722c4837d604bda6f684d24eb73c on its own branch. That way we only get the feature (and nothing else) in the diff for your PR.